pyproject.toml

```diff
@@ -46,6 +46,7 @@ grpcio = {version=">=1.49.0,<2.0", optional = true}
 protobuf = { version = "^4.21", optional = true }
 hightime = "^0.2.1"
 tzlocal = "^5.0"
+python-decouple = "^3.8"
```

Comment:
Use `>=` for this one. I am not as worried about `python-decouple` breaking source compatibility as I am about `grpcio`, `protobuf`, and `tzlocal`.